setup.cfg

```diff
@@ -1,6 +1,4 @@
 [flake8]
 max-line-length = 120
```

Comment:
```suggestion
max-line-length = 88
```
This should work as we use black since https://github.com/python-social-auth/social-app-django/pull/394.

Reply:
Anyway merging as is now, this could be done in another PR.